assets/src/components/form/inputs/date-time-input.js

```diff
@@ -3,10 +3,9 @@
  */
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
-import { Component } from '@wordpress/element';
+import { useState, useEffect } from '@wordpress/element';
```

Comment:
this line is fine but there is some existing code in the "External imports" that should be moved down under an "Internal imports" comment heading.
ie: this existing code should be moved:

```jsx
import {
	EspressoIcon,
	ESPRESSO_ICON_CALENDAR
} from '../../ui/image/espresso-icon';
```

can you also do that as part of this PR plz?

Reply:
Done :)

Reply:
I see you added an "Internal dependencies" heading, but those imports above from from `'../../ui/image/espresso-icon'` are still listed under the "External imports".

Can you move those as well plz?